.travis.yml

```diff
@@ -28,6 +28,8 @@ before_install:
   - mysql --version
   - mysql -e 'CREATE DATABASE AccountMetadata;'
   - mysql -e 'USE AccountMetadata; SOURCE ./ambry-account/src/main/resources/AccountSchema.ddl;'
+  - mysql -e 'CREATE DATABASE AmbryBlobs;'
```

Comment:
I would have preferred AmbryNamedBlobs, but you may have already provisioned it with that name.

Reply:
I can change this. This is really just the name for the integration test DB and does need to match the naming patterns in other places.